### PR TITLE
IDEMPIERE-6316: Wrong Name on RecordInfo panel when the value of the …

### PR DESCRIPTION
**Pull request by nmicoud**
…column is Context

https://idempiere.atlassian.net/browse/IDEMPIERE-6316

# Pull Request Checklist

- [X] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [X] My code follows the best practices of this project
- [X] I have performed a self-review of my own code
- [X] My code is easy to understand and review. 
- [X] I have checked my code and corrected any misspellings
- [ ] In hard-to-understand areas, I have commented my code.
- [X] My changes generate no new warnings
### Tests
- [X] I have tested the direct scenario that my code is solving
- [ ] I checked all collaterals that can be affected by my changes, and tested other potential affected scenarios
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added unit tests that prove my fix is effective or that my feature works
### Documentation
- [ ] I have made corresponding changes to the documentation as follows:
- - [ ] New feature (non-breaking change which adds functionality): I have created the New Feature page in the project wiki explaining the functionality and how to use it. If relevant, I have committed sample data to the core seed to have usable examples in GardenWorld.
- - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected): I have documented the change in a clear way that everyone in the community can understand the impact of the change.
- - [ ] Improvement (improves and existing functionality): This documentation is needed if the improvement changes the way the user interacts with the system or the outcome of a process/task changes. If it is just, for instance, a performance improvement, documentation might not be needed. 
- [ ] The changed/added documentation is in the project wiki (not privately-hosted pdf files or links pointing to a company website) and is complete and self-explanatory.

### CODE REVIEW

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WRecordInfo.java [hunk 1]
```diff
@@ -481,7 +481,7 @@ private void addLine (int AD_Column_ID, Timestamp Updated, int UpdatedBy,
 		Vector<String> line = new Vector<String>();
 		//	Column
 		MColumn column = MColumn.get (Env.getCtx(), AD_Column_ID);
-		line.add(Msg.translate(Env.getCtx(), column.getColumnName()));
+		line.add(Msg.getElement(Env.getCtx(), column.getColumnName()));
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WRecordInfo.java:484]: I think we should replace other `translate` in this class with `getElement` call as well.

**Review by hengsin — CHANGES_REQUESTED** [review]

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WRecordInfo.java (hunk 1 above)
Comment (nmicoud) [reply org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/window/WRecordInfo.java:484]: done

**Comment by CarlosRuiz-globalqss** [comment]
Hi @nmicoud and @hengsin 

I have seen this problem before in other parts of the code.

What if change the translate method to look for element when the message is not found?

**Comment by hengsin** [comment]
Msg.translate check context, follow by ad_message and finally ad_element. I think this is wrong for this particular use case since translation of column name should comes from ad_element, not ad_message or context. There's another issue need checking here - the IsSOTrx property. The current getElement call means IsSOTrx is hard coded to true and I think that's not right for some window (for e.g the Purchase Order window).

**Comment by CarlosRuiz-globalqss** [comment]
Hi, the following patch solves the problem of IsSOTrx mentioned by @hengsin :
[pr2560PR0.patch.txt](https://github.com/user-attachments/files/17918036/pr2560PR0.patch.txt)

I added a comment in the ticket for further discussion as this problem seems to be bigger than what is fixed here:
https://idempiere.atlassian.net/browse/IDEMPIERE-6316?focusedCommentId=51315

**Review by hengsin — APPROVED** [review]
looks good

**Pull request merged**